src/js/entry/web.js

```diff
@@ -190,7 +190,7 @@ function Config ($routeProvider, $injector) {
   });
 
   $routeProvider.otherwise({redirectTo: '/404'});
-};
+}
```

Comment:
Because 1. jshint complains about it, or actually because 2. the brace is to close a function definition so the semicolon isn't needed.